UnitTests/TransportTests.cpp

```diff
@@ -18,6 +19,8 @@
 
 BOOST_AUTO_TEST_SUITE(TransportTests)
 
+using Traits = IPC::UnitTest::Mocks::NullTimeoutTraits;

```

Comment:
Well, timeout might not be the only thing which we may want to mock in future changes. It just happen to be the only one here.